### PR TITLE
Use mongoose to connect server to MongoDB database hosted on MLab

### PR DESCRIPTION
**Pull request by charliearlie**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,12 +15,17 @@
...
     "debug": "~2.6.9",
+    "dotenv": "^6.0.0",
     "ejs": "^2.6.1",
     "express": "~4.16.0",
+    "express-session": "^1.15.6",
+    "express-validator": "^5.3.0",
     "http-errors": "~1.6.2",
+    "mongoose": "^5.3.2",
     "morgan": "~1.9.0",
     "pug": "2.0.0-beta11"
```
Comment (LorenzoKoundouris) [inline package.json]: It's not part of this PR but if `pug` isn't being used we should remove

File: app.js [hunk 2]
```diff
@@ -1,30 +1,52 @@
 var createError = require('http-errors');
 var express = require('express');
+const session = require('express-session');
+const mongoose = require('mongoose');
+const MongoStore = require('connect-mongo')(session);
```
[the remark this answers is not in the document]
Comment (charliearlie) [reply app.js]: Nope. That's the convention.

[the remark this answers is not in the document]
Comment (charliearlie) [reply app.js]: https://github.com/jdesboeufs/connect-mongo
